helm-chart/Chart.yaml

```diff
@@ -0,0 +1,17 @@
+apiVersion: v2
+name: avatar
+description: A Helm chart for the Avatar platform
+type: application
+
+# This is the chart version. This version number should be incremented each time you make changes
+# to the chart and its templates, including the app version.
+# Versions are expected to follow Semantic Versioning (https://semver.org/)
+version: 0.1.2
+
+# This is the version number of the application being deployed. This version number should be
+# incremented each time you make changes to the application. Versions are not expected to
+# follow Semantic Versioning. They should reflect the version the application is using.
+# It is recommended to use it with quotes.
+appVersion: "0.4.5"
```

Comment:
This is going to be difficult to maintain

Reply:
Agreed.